content/network/_index.md

```diff
@@ -132,6 +133,96 @@ it isn't necessary to publish the container's ports.
 You can enable inter-container communication by connecting the containers to the
 same network, usually a [bridge network](./drivers/bridge.md).
 
+If no host IP is given in a port mapping, the bridge network is IPv4-only,
+and `--userland-proxy=true`; ports on the Docker host's IPv6 addresses will
+be mapped to the container's IPv4 address.
```

Comment:
Might just be me, but I have a hard time parsing this sentence. Is it saying that if all of these three conditions are true, then IPv6 addresses on the host map to IPv4 addresses of containers? What about something like this, where it leads with the main clause / outcome:

```suggestion
Ports on the host's IPv6 addresses will map to the container's IPv4 address
if no host IP is given in a port mapping, the bridge network is IPv4-only,
and `--userland-proxy=true` (default).
```

Reply:
Much better, thank you.